src/mngt/ocf_mngt_cache.c

```diff
@@ -2008,6 +2008,8 @@ static void _ocf_mngt_load_unsafe_complete(void *priv, int error)
 				-OCF_ERR_START_CACHE_FAIL);
 	}
 
+	context->metadata.shutdown_status = ocf_metadata_dirty_shutdown;
```

Comment:
That's true. Do you think this value should be set is some other place in code?

Reply:
In standby pipeline we call `_ocf_mngt_init_properties()` which sets shutdown status to clean. Maybe we should replace this step with standby-specific one, that sets it already to dirty instead of toggling this status every second step ;)